build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.0'
```

Comment:
1) Changing the android gradle plugin version should not be a part of this change. A commit should only target one change at a time, so that if something breaks, we know what caused that.